components/style/properties/declaration_block.rs

```diff
@@ -935,13 +935,23 @@ impl<'a, 'b, 'i> AtRuleParser<'i> for PropertyDeclarationParser<'a, 'b> {
     type Error = SelectorParseError<'i, StyleParseError<'i>>;
 }
 
+fn is_non_mozilla_vendor_identifier(name: &str) -> bool {
+    (name.starts_with("-") && !name.starts_with("-moz")) ||
```

Comment:
I guess at least the `-moz` check should be ascii case insensitive: You can use `style::str::starts_with_ignore_ascii_case` for that.